plugins/postcss-nesting/src/index.ts

```diff
@@ -6,13 +6,16 @@ import walk from './lib/walk.js';
 export type pluginOptions = {
 	/** Avoid the `:is()` pseudo class as much as possible. default: false */
 	noIsPseudoSelector?: boolean,
+	/** Silence the `@nest` warning. */
+	silenceAtNestWarning?: boolean,
```

Comment:
Might also be worth to surface on the CHANGELOG? Users seem to look for these

Reply:
Good catch! Thank you 🙇 

I've added it in the CHANGELOG and general README.
I also linked to the Stylelint plugin.